Gotrue/Api.cs

```diff
@@ -89,10 +89,19 @@ public Task<Session> SignInWithEmail(string email, string password)
         /// Sends a magic login link to an email address.
         /// </summary>
         /// <param name="email"></param>
+        /// <param name="options"></param>
         /// <returns></returns>
-        public Task<BaseResponse> SendMagicLinkEmail(string email)
+        public Task<BaseResponse> SendMagicLinkEmail(string email, SignInOptions options = null)
         {
             var data = new Dictionary<string, string> { { "email", email } };
+            if (options != null)
+            {
+                if (!string.IsNullOrEmpty(options.RedirectTo))
+                {
+                    data.Add("redirect_to", options.RedirectTo);
```

Comment:
I'm not sure if we need to uri encode the url at this point or if its handled by HttpUtility in Helpers.MakeRequest.

When we have the tests working, we can test this.